src/main/com/leandro/it/FibonacciFunction.java

```diff
@@ -0,0 +1,25 @@
+package com.leandro.it;
+
+import exceptions.InvalidNumberException;
+
+public class FibonacciFunction {
+
+    int n;
+
+    public FibonacciFunction() {
```

Comment:
why declare an empty constructor? it will be created by java if non exist other. In other words it's innecesary